doc/manual/install-domserver.rst

```diff
@@ -179,6 +179,20 @@ this approach is that the webserver is not aware of the actual client IP. This
 means that access logs for the webserver will still report the IP of the proxy
 or loadbalancer.
 
+Scaling the domserver
+---------------------
+
+Documenting all possible ways to properly size the domserver is out of scope for this
+manual. But properly think about the memory management for the PHP-FPM children with
+~40 per gigabyte of memory, 500 for a 16Gb system.
```

Comment:
This is not clearly phrased. Does it mean to say that 40 children consume 1 GB of memory? Is this a suggestion for how many to configure?

Reply:
Its copied from the php pool snippet, and it should say something about `total amount of memory in gigabytes divided by 40`, I fully agree its not clearly phrased but I can't think of a better fix which also reads natural.